src/silx/gui/plot/PlotWidget.py

```diff
@@ -2298,15 +2298,13 @@ def setActiveCurveSelectionMode(self, mode: Literal["legacy", "atmostone", "none
     def getActiveCurveSelectionMode(self) -> Literal["legacy", "atmostone", "none"]:
         """Returns the current selection mode.
 
-        It can be "atmostone", "legacy" or "none".
+        It can be "atmostone" (at most one), "legacy" or "none".
 
         :rtype: str
         """
         return self._activeCurveSelectionMode
 
-    def getActiveImage(
-        self, just_legend: bool = False
-    ) -> items.ImageData | items.ImageRgba | str | None:
+    def getActiveImage(self, just_legend: bool = False) -> items.ImageBase | str | None:
```

Comment:
Used the same type as returned by `addImage`, it can also be an ImageComplex for example

Reply:
:sweat_smile: 

That's also inline with: `def _getActiveItem(...) -> items.Curve | items.Scatter | items.ImageBase | None:`